Dockerfile

```diff
@@ -103,3 +103,4 @@ COPY --chown=1001:101 $APP_PATH /app/samvera/hyrax-webapp
 COPY --chown=1001:101 . /app/samvera/hyrax-engine
 
 RUN bundle install --jobs "$(nproc)"
+RUN RAILS_ENV=production SECRET_KEY_BASE='fakesecret1234' DB_ADAPTER=nulldb DATABASE_URL='postgresql://fake' bundle exec rake assets:precompile
```

Comment:
is this needed? it looks like it duplicates asset precompilation in the worker, which should have just run due to the `ONBUILD` at https://github.com/samvera/hyrax/pull/5089/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R76.